stats/stats.go

```diff
@@ -59,6 +59,27 @@ func (s *Begin) IsClient() bool { return s.Client }
 
 func (s *Begin) isRPCStats() {}
 
+// InitialResolverResult represents an event that the resolved finished
+// resolving if the ClientConn blocks on resolver resolution while performing a
+// RPC.
+type InitialResolverResult struct{}
+
+// IsClient indicates if the stats information is from client side. Only Client
+// Side interfaces with a resolver, thus always returns true.
+func (irr *InitialResolverResult) IsClient() bool { return true }
```

Comment:
Minor nit for all these methods. Since the receiver is not used, we don't even have to name it. So, this can simply be:
```golang
func (*InitialResolverResult) IsClient() bool { return true }
```

Reply:
Hmmmmm ok I was actually wondering that too. Deleted, thanks.